packages/datasource/src/api/blog/index.js

```diff
@@ -10,7 +10,10 @@ export const getRecommend = (params) => blogClient.get(`/api/posts/recommend`, {
 
 export const getPosts = (params) => blogClient.get(`/api/posts`, { params });
 //TODO: need to pass header of X-Forward-For for forward client real ip to backend
-export const getPostBySlug = (slug) => blogClient.get(`/api/posts/${slug}/detail`);
+export const getPostBySlug = (slug, ip) =>
+  blogClient.get(`/api/posts/${slug}/detail${ip !== undefined ? '?visit=true' : ''}`, {
+    headers: { 'X-Forwarded-For': ip },
```

Comment:
You need to force this header only added by next server but not the client

Reply:
> You need to force this header only added by next server but not the client

OK